Source/WebCore/rendering/RenderView.cpp

```diff
@@ -455,7 +455,7 @@ void RenderView::paintBoxDecorations(PaintInfo& paintInfo, const LayoutPoint&)
     if (RenderElement* rootRenderer = documentElement ? documentElement->renderer() : nullptr) {
         // The document element's renderer is currently forced to be a block, but may not always be.
         auto* rootBox = dynamicDowncast<RenderBox>(*rootRenderer);
-        rootFillsViewport = rootBox && !rootBox->x() && !rootBox->y() && rootBox->width() >= width() && rootBox->height() >= height();
+        rootFillsViewport = rootBox && (!rootBox->x() && !rootBox->y() && rootBox->width() >= width() && rootBox->height() >= height()) && !rootBox->hasClipPath();
```

Comment:
What are the implications for non-SVG renderers? Shouldn't CSS have the same issue for clip paths?

Reply:
It looks like SVG is special, see this comment in RenderView.cpp:
// SVG documents and XML documents with SVG root nodes are transparent.

Reply:
Fair enough, but you are not checking specifically for SVG?

Reply:
I am in the new patch :)